src/sql/parts/notebook/notebookActions.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+ import * as sqlops from 'sqlops';
+
+import { Action } from 'vs/base/common/actions';
+import { TPromise } from 'vs/base/common/winjs.base';
+import { localize } from 'vs/nls';
+import { IContextViewProvider } from 'vs/base/browser/ui/contextview/contextview';
+
+import { SelectBox } from 'sql/base/browser/ui/selectBox/selectBox';
+import { INotebookModel } from 'sql/parts/notebook/models/modelInterfaces';
+
+const msgLoading = localize('loading', 'Loading kernels...');
+export class AddCellAction extends Action {
+	public static ID = 'jobaction.notebookTest';
```

Comment:
This ID is not very useful and looks like it was copied from the jobs framework. Could we use `notebook.addCell` here?